user/tests/wiring/ble/heartrate/application.cpp

```diff
@@ -20,21 +20,16 @@
 /* Includes ------------------------------------------------------------------*/
 #include "application.h"
 
-BLEAttribute heartrate(READ | NOTIFY, "heartrate");
-BLEConnection* myConn;
+BLEAttribute heartrate("heartrate", READ | NOTIFY);
+int newHr = 1234;
 
 void setup() {
-    BLE.on();
-
     BLE.advertise();
-
-    BLEDevice local;
-    myConn = BLE.connect();

```

Comment:
There should be no need to call `connect()` here.
`advertise()` should set up everything that is required.